docs/meetup.md

```diff
@@ -1,25 +1,24 @@
 # 定期ミートアップ
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/zA_Hy9E0cp4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/hPGFiUp5s_A?start=673" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
The video starts from 673s. Is it intentional?

Reply:
Yes, I intended to display an attractive preview of the YouTube channel by setting a certain time point in the video, but it didn't make sense since the preview was fixed to the start of the video.
Alternatively, I set the appropriate preview in YouTube config.